Mention defusedxml and olefile in installation instructions (#7509)
python-pillow/Pillow: docs/installation.rst

    python3 -m pip install --upgrade pip
    python3 -m pip install --upgrade Pillow

> Optionally, install :pypi:`olefile` for Pillow to read FPX and MIC images::
> 
>     python3 -m pip install --upgrade olefile

Reviewer: Perhaps also mention `defusedxml` here?

```suggestion
Optionally, install :pypi:`defusedxml` for Pillow to read XMP data,
and :pypi:`olefile` for Pillow to read FPX and MIC images::

    python3 -m pip install --upgrade defusedxml olefile
```

Author: Good point!

    python3 -m pip install --upgrade pip
    python3 -m pip install --upgrade Pillow

> :pypi:`olefile` can additionally be installed to allow Pillow to read FPX and
> MIC images::

Reviewer 2: With "active voice" instead of "passive":
```suggestion
Optionally, install :pypi:`olefile` for Pillow to read FPX and MIC images::
```
```
